src/tests/ftest/util/pool_security_test_base.py

```diff
@@ -346,7 +317,12 @@ def verify_pool_readwrite(self, uuid, action, expect='Pass'):
         if action.lower() == "write":
             result = daos_cmd.container_create(pool=uuid)
         elif action.lower() == "read":
-            result = daos_cmd.pool_query(pool=uuid)
+            json_result = daos_cmd.pool_query(pool=uuid)
```

Comment:
An alternative to consider since `verify_daos_pool_cont_result()` only checks the `.exit_status` and `.stdout/.stderr` for error messages:
```
            # pylint: disable=protected-access
            result = daos_cmd._get_result(("pool", "query"), pool=uuid)
```